### PR TITLE
Not native, but text

### PR DESCRIPTION
**Pull request by bcoca**
fix incorrect usages of to_native

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

all

### CODE REVIEW

**Comment by rambleraptor** [comment]
Looks fine on the `gcp_compute` plugin.

**Review by mkrizek — APPROVED** [review]

**Comment by mkrizek** [comment]
Just needs adding missing `to_text` imports and removing unused `to_native` ones.

File: lib/ansible/plugins/inventory/gcp_compute.py [hunk 1]
```diff
@@ -120,16 +120,10 @@
...
+from ansible.module_utils._text import to_text
 from ansible.module_utils.gcp_utils import GcpSession, navigate_hash, GcpRequestException
 from ansible.plugins.inventory import BaseInventoryPlugin, Constructable, Cacheable
 
-try:
-    import google.auth
-    import requests
-except ImportError:
-    raise AnsibleError('The gcp dynamic inventory plugin requires the requests and google-auth libraries')
-
```
Comment (sivel) [inline lib/ansible/plugins/inventory/gcp_compute.py:132]: Removing this changes the behavior. Even though there weren't used directly, it was used to determine if the correct libs were installed.

You could instead add `from ansible.module_utils.gcp_utils import HAS_GOOGLE_LIBRARIES`, but I believe raising the error is still useful.

Comment (bcoca) [reply lib/ansible/plugins/inventory/gcp_compute.py:132]: should still not be raised on import, but on parse

**Review by sivel — APPROVED** [review]

**Pull request merged**